tests/BrowserKitConfig.php

```diff
@@ -17,7 +18,7 @@ public static function getInstance()
      */
     public function createDriver()
     {
-        $client = new Client(require(__DIR__.'/app.php'));
+        $client = new Client(new FixturesKernel());
```

Comment:
This answers my question on test suite PR on how `FixturesKernel` would be used 😄